Test1/Test1/ThreadSafePriorityQueue.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Test1;
+
+public class ThreadSafePriorityQueue<P, V > where P : IComparable<P>
```

Comment:
```suggestion
public class ThreadSafePriorityQueue<Priority, Value> where Priority : IComparable<P>
```
Во-первых, форматирование.
Во-вторых, имена типовых переменных - это имена. Поэтому они должны быть содержательными.

Reply:
Форматирование: -0.5.
Имена типовых переменных: -0.5.